docs/management/api/suspend-process-api.asciidoc

```diff
@@ -0,0 +1,92 @@
+[[suspend-process-api]]
+=== Suspend a process
+
+Suspend a process on a host running {endpoint-cloud-sec}.
+
+You must have the `superuser` role to perform this action and at least an Enterprise license.
+
+==== Request URL
+
+`POST <kibana host>:<port>/api/endpoint/action/suspend_process`
+
+==== Request body
+
+A JSON object with these fields:
+
+[width="100%",options="header"]
+|==============================================
+|Name |Type |Description |Required
+
+|`endpoint_ids` |Array (String) |The IDs of each endpoint you want to issue this action to. |Yes
```

Comment:
```suggestion
|`endpoint_ids` |Array (String) |The ID(s) of endpoint(s) you want to issue this action to. |Yes
```
Assuming it could be just one endpoint, this applies elsewhere in the PR as well.